### DISCUSSION TITLE
Unable to use Disclosure tag in webpack module federation architecture.

### DISCUSSION THREAD

**Discussion by OdaTech01** [tailwindlabs/headlessui#3103] [Help]
Hi Team,

I am using headlessui in my app which is having module federation architecture. 

It is working fine in my remote app. But when I using remote app component in host app getting below error. Need help from you.

![image](https://github.com/tailwindlabs/headlessui/assets/166930865/674e5a4a-581e-4bbc-854e-38f5b5bef648)

**Comment by RobinMalfait** [comment]
Hey!

This is not a bug we can solve in Headless UI itself. The error you see typically means that you are loading multiple versions of React at the same time.

Can you share a minimal reproduction repo?

**Reply by OdaTech01** [reply]
Hi @RobinMalfait ,

Thank you for your quick response. Please find below reproduction repo and please let me know if 
you need any further details.

https://github.com/OdaTech01/Tailwindcss/tree/main/microfrontend-container

Thanks.
